### DISCUSSION TITLE
What alternatives could potentially serve as replacements for Azure Media Services?

### DISCUSSION THREAD

**Discussion by brindabindu** [ant-media/Ant-Media-Server#5834] [General]
Hi,
Currently in the process of exploring new platforms to host my videos and migrate data from Azure Media Services. The recent retirement announcement has prompted me to seek alternatives. I'm keen on finding a solution that offers seamless migration, scalability, and robust features for my media content. If anyone has recommendations or experiences to share, I'd greatly appreciate the insights!"

**Comment by USAMAWIZARD** [comment]
Hi @brindabindu ,
I hope you are doing well , As you are looking to migrate from Azure Media Services , I would like to introduce You to Ant Media Server. Its a very feature rich  media server which is highly  Scalable and provides Adaptive Streaming and Ultra Low Latency Streaming you can learn more about it [here](https://antmedia.io/).

We also have many other clients who are migrating to us from Azure Media Service.I'd love to learn more about your specific requirements and discuss how Ant Media can cater to your video streaming needs. We can arrange a personalized demo or answer any questions you might have.Please feel free to book a meeting with me [here](https://calendly.com/usama-tahseen).

BTW we are also running Black Friday Discount which is valid till end of the month.



Regards,
Usama

**Comment by USAMAWIZARD** [comment]
@brindabindu Please let me know if you are still interested , I will be here to help.

**Comment by dwarak17** [comment]
Hi USAMAWIZARD,

Thanks for your help, but after AMS [retirement](https://www.vplayed.com/blog/azure-media-services-retirement/), I approached VPlayed, and the pricing and their support are good. here

**Reply by USAMAWIZARD** [reply]
Ant Media Also has a flexible pricing and very responsive support team , let us know if you want to have a more salable and robust solution in the future.
